boards/arm/cc3235sf_launchxl/doc/index.rst

```diff
@@ -0,0 +1,305 @@
++-----------+------------+-----------------------+
+| SPI_0     | on-chip    | WiFi host driver      |
++-----------+------------+-----------------------+
+
+.. note::
+
+   For consistency with TI SimpleLink SDK and BoosterPack examples,
+   the I2C driver defaults to I2C_BITRATE_FAST mode (400 kHz) bus speed
+   on bootup.
+
+The accelerometer, temperature sensors, or other peripherals
+accessible through the BoosterPack, are not currently supported.
+
+Connections and IOs
+====================
+
+Peripherals on the CC3235SF LaunchXL are mapped to the following pins in
+the file :zephyr_file:`boards/arm/cc3235sf_launchxl/pinmux.c`.
+
++------------+-------+-------+
+| Function   | PIN   | GPIO  |
++============+=======+=======+
+| UART0_TX   | 55    | N/A   |
++------------+-------+-------+
+| UART0_RX   | 57    | N/A   |
++------------+-------+-------+
+| LED D7 (R) | 64    | 9     |
++------------+-------+-------+
+| LED D6 (O) | 01    | 10    |
++------------+-------+-------+
+| LED D5 (G) | 02    | 11    |
++------------+-------+-------+
+| Switch SW2 | 15    | 22    |
++------------+-------+-------+
+| Switch SW3 | 04    | 13    |
++------------+-------+-------+
+
+The default configuration can be found in the Kconfig file at
+:zephyr_file:`boards/arm/cc3235sf_launchxl/cc3235sf_launchxl_defconfig`.
+
+
+Programming and Debugging
+*************************
+
+TI officially supports development on the CC3235SF using the TI
+`CC32xx SDK`_ on Windows and Linux using TI tools: Code Composer
+Studio for debugging and `UniFlash`_ for flashing.
+
+For Windows developers, see the `CC32xx Quick Start Guide`_ for
+instructions on installation of tools, and how to flash the board using
+UniFlash.
+
+Note that zephyr.bin produced by the Zephyr SDK may not load via
+UniFlash tool.  If encountering difficulties, use the zephyr.elf
+file and openocd instead (see below).
+
+The following instructions are geared towards Linux developers who
+prefer command line tools to an IDE.
+
+Before flashing and debugging the board, there are a few one-time board
+setup steps to follow.
+
+Prerequisites:
+==============
+
+#. Download and install the latest version of `UniFlash`_.
+#. Jumper SOP[2..0] (J15) to [010], and connect the USB cable to the PC.
+
+   This should result in a new device "Texas Instruments XDS110 Embed
+   with CMSIS-DAP" appearing at /dev/ttyACM1 and /dev/ttyACM0.
+
+#. Update the service pack, and place the board in "Development Mode".
+
+   Setting "Development Mode" enables the JTAG interface, necessary
+   for subsequent use of OpenOCD and updating XDS110 firmware.
+
+   Follow the instructions in Section 2.4 "Download the Application",
+   in the `CC32xx Quick Start Guide`_, except for steps 5 and 6 in
+   Section 2.4.1 which select an MCU image.
+
+#. Ensure the XDS-110 emulation firmware is updated.
+
+   Download and install the latest `XDS-110 emulation package`_.
+
+   Follow the directions here to update the firmware:
```

Comment:
This would read better as:
```
   Follow these `xds110 firmware update directions
   <http://software-dl.ti.com/ccs/esd/documents/xdsdebugprobes/emu_xds110.html#updating-the-xds110-firmware>`_
```